intelmq/lib/bot.py

```diff
@@ -394,5 +399,56 @@ def __load_harmonization_configuration(self):
                             "Key %s is not valid." % _key)
 
 
+class ParserBot(Bot):
+
+    def parse_csv(self, report):
```

Comment:
We need to be careful in having too many nested subclasses... It could lead to low code readability... but I agree this may not be the best place for this... maybe some kind of util package where we could have a few parse functions that several bots use?